docs/library/ssl.rst

```diff
@@ -31,6 +31,13 @@ Functions
      until it completes. Note that in AXTLS the handshake can be deferred until the first
      read or write but it then blocks until completion.
 
+   - *cert_reqs* determines whether the peer (server or client) must present a valid certificate.
+     Note that for mbedtls based ports, ``ssl.CERT_NONE`` and ``ssl.CERT_OPTIONAL`` will not
+     validate any certificate, only ``ssl.CERT_REQUIRED`` will.
+
+   - *cadata* is the CA certificate chain that will validate the peer's certificate.
```

Comment:
This takes a concatenation of DER-format certificates, right?  If so would be good to document that here.

Reply:
Done!